s3/pkg/datastore/yig/utils/global_id.go

```diff
@@ -0,0 +1,74 @@
+package utils
```

Comment:
Hi @hopkings2008  Please add a license announcement at the beginning. At the same time, add some notes to explain it used the snowflake. Thanks.

Reply:
thanks @sfzeng for your suggestion, i will add that.